packages/test/stochastic-test-utils/src/xsadd.ts

```diff
@@ -5,7 +5,17 @@
 
 /* eslint-disable no-bitwise */
 
-import { RandomCtor, Random } from "best-random";
+import { Random } from "best-random";
+
+// Perf: We avoid the use of an ES6 'class' for a modest performance gain, but allow the use
+//       of the 'new' keyword using a ctor interface (node 12? x64).
+
+/**
+ * Construct a new instance of the XSadd random number generator, seeding it with up to
+ * four 32b integers.  If no seeds are provided, the PRNG is non-deterministically seeded
+ * using Math.random().
+ */
+export type XSaddCtor = new (seed0?: number, seed1?: number, seed2?: number, seed3?: number) => Random;
```

Comment:
It was a coincidence (not a contract) that XSadd and 'best-random's ctor both accept 128b seeds as uint32x4.